frontends/concrete-python/scripts/checks/checks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+sudo apt install -y graphviz libgraphviz-dev
+cd frontends/concrete-python
+make venv
+source .venv/bin/activate
+make pcc
+make check_links
```

Comment:
Why not add this to `pcc`, it's better to just run `make pcc` locally before creating a PR.

Reply:
ok done